.gitlab-ci.yml

```diff
@@ -3,10 +3,28 @@ stages:
   - unit-tests
   - dev
   - promote-to-stg
```

Comment:
(Github won't let me add this comment where I want it.)

Should we have `common-promote-to-stg` and `common-promote-to-prd` Jobs?